src/platform_impl/linux/window.rs

```diff
@@ -569,9 +575,12 @@ impl Window {
   }
 
   pub fn set_maximized(&self, maximized: bool) {
+    let resizable = self.is_resizable();
```

Comment:
instead of just ignoring the maximized request, should we instead make the window resizable, maximize it, then make it non-resizable again?

Reply:
In the event loop, using window.set_resizable(true); followed immediately by window.maximize(); did not work. However, adding a delay of 0.1 seconds between the two calls made it work.

```rust
      WindowRequest::Maximized(maximized, resizable) => {
            if maximized {
              window.set_resizable(true);

              let window_clone = window.clone();
              glib::timeout_add_local(std::time::Duration::from_millis(100), move || {
                window_clone.maximize();
                glib::ControlFlow::Break
              });

              let window_clone = window.clone();
              glib::timeout_add_local(std::time::Duration::from_millis(200), move || {
                window_clone.set_resizable(resizable);
                glib::ControlFlow::Break
              });
            } else {
              window.unmaximize();
            }
          }
```

While it’s not precise, in Wayland, a gap between events is necessary when committing a buffer.

Reply:
I noticed that processing multiple events simultaneously causes conflicts. 
I modified the code to use `WindowMaximizeProcess` to handle `resizable` and `maximize` sequentially when creating a new window and during the `WindowRequest::Maximized` event.